docs/source/casts.rst

```diff
@@ -34,8 +34,8 @@ quite understand what is going on.
           assert isinstance(o, int)
           print(o + 5)  # OK: type of 'o' is 'int' here
 
-You don't need a cast for expressions with type ``Any``, or when
-assigning to a variable with type ``Any``, as was explained earlier.
+You don't need a cast for expressions with type :py:data:`~typing.Any`,
```

Comment:
Again, `Any` should probably not be linked. It's just too common. Also, the [explanation of this type in the Python docs](https://docs.python.org/3/library/typing.html#typing.Any) is rather lame. If anything, it should probably link to an explanation of the concept in the mypy docs (if there is one), or perhaps in [PEP 483](https://www.python.org/dev/peps/pep-0483/#id8).